src/java/my/example/Test.java

```diff
@@ -0,0 +1,51 @@
+package my.example;
+
+//Урок №2 вправи 1-3
+
+public class Test {
+    public static void main(String[] args) {
+      //int
+      int int10 = 3;
+      int int16 = 0x4;
+      int int2 = 0b100;
+      int sumInt = int2+int10+int16;
```

Comment:
Треба додавати пробіли біля операторів  
```suggestion
      int sumInt = int2 + int10 + int16;
```